.travis.yml

```diff
@@ -80,9 +80,6 @@ jobs:
       # Run Kubernetes Job and run Auth (May not need keystore chart)
       - helm install --name keystore services-bc-mp/keystore
       - helm install --name auth services-bc-mp/auth
```

Comment:
this line must be changed to point to the auth repo because the chart is no longer located on the main repo